queries/highlights.scm

```diff
@@ -78,6 +78,8 @@
   "global"
   "runtime"
   "wincmd"
+  "break"
+  "continue"
```

Comment:
I wonder if these do not belong to the `repeat` capture in the end...
Your opinion?

Reply:
Indeed, `for`, `while`, `break`, `continue`,  and friends are `@repeat` in every other language on nvim-treesitter.

Reply:
Let's survey what other tree-sitter query files do. I ran `vimgrep '\v<(break|continue)_?>\_.{-}\@\S+' queries/*/highlights.scm` on the nvim-treesitter repo ([as of commit 0fc45ea](https://github.com/nvim-treesitter/nvim-treesitter/tree/0fc45ea650df87c47ab4fcfe903693ce170c71c3)) and here's what I found:

10 `highlights.scm` capture both in `@repeat` (C, D, gdscript, Hack, Julia, Pascal, Python, Rust, Swift, WGSL);
1 captures `break` in `@conditional` and `continue` in `@repeat` (C#);
1 captures `break` in `@exception` and `continue` in `@repeat` (Dart);
2 capture `break` in `@keyword` and `continue` in `@repeat` (ECMAScript, PHP);
6 capture both in `@keyword` (Fish, Fortran, Go, Java, R, Ruby);
5 capture `break` in `@keyword` (Lua, Teal, Tiger, V, Verilog);
1 captures both in `@conditional` (solidity);
1 captures both in `@keyword.return` (Zig).

So the most common thing is to capture them in `@repeat`.  Do you want me to move the keywords there?

Reply:
1. With my latest force push, `break` and `continue` are now captured in `@repeat`.

Reply:
The conclusion was that languages should distinuguish between `break` as `@conditional` when used in `switch` statement and `@repeat` when used in loops

Reply:
Interesting, although this doesn't change anything in our case since Vimscript doesn't have `switch` statements or an equivalent; thankfully, we don't have to handle this here.